modules/migration-mtc-release-notes-1-8-1.adoc

```diff
@@ -23,4 +23,8 @@ For more details, see link:https://access.redhat.com/security/cve/cve-2023-39325
 [id="known-issues-1-8-1_{context}"]
 == Known issues
 
-There are no major known issues in this release.
+{mtc-full} ({mtc-short}) 1.8.1 has the following known issues:
+
+.Ansible Operator is broken when {VirtProductName} is installed
```

Comment:
@rayfordj & @stillalearner - I have included this as a resolved issue in 1.8.4 and as a known issue in < 1.8.4
I do not really have anywhere more applicable in the documentation 
Is it ok in < 1.8.4 && >= 1.8.0 as a resolved issue?
Easy enough to remove